hack/e2e-debug.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source $(dirname "$0")/../test/e2e-common.sh
```

Comment:
The other test files seem to do this before sourcing e2e-common, so for consistency let's also set the env var before sourcing in this file.
```suggestion
export GO111MODULE=on

source $(dirname "$0")/../test/e2e-common.sh
```